examples/simulation-tensorflow-tabular/README.md

```diff
@@ -0,0 +1,98 @@
+# Flower Simulation example using TensorFlow/Keras in tabular data
+
+This introductory example uses the simulation capabilities of Flower to simulate a large number of clients on a single machine. Take a look at the [Documentation](https://flower.ai/docs/framework/how-to-run-simulations.html) for a deep dive into how Flower simulation works. This example uses [Hugging Face](https://huggingface.co/datasets/julien-c/titanic-survival) to download, [Flower Datasets](https://flower.ai/docs/datasets/)  to partition and preprocess the Titanic dataset. This examples uses 5 clients by default.
+
+## Running the example (via Jupyter Notebook)
+
+Run the example on Google Colab: [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/damjimenezgu/flower/blob/simulation-tensorflow-tabular/examples/simulation-tensorflow-tabular/sim.ipynb)
+
+Alternatively, you can run `sim.ipynb` locally or in any other Jupyter environment.
+
+## Running the example
+
+Start by cloning the code example. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
+
+```shell
+git clone --depth=1 https://github.com/adap/flower.git && mv flower/examples/simulation-tensorflow-tabular . && rm -rf flower && cd simulation-tensorflow-tabular
+```
+
+This will create a new directory called `simulation-tensorflow` containing the following files:
+
+```
+-- README.md       <- Your're reading this right now
+-- sim.ipynb       <- Example notebook
+-- sim.py          <- Example code
+-- pyproject.toml  <- Example dependencies
+-- requirements.txt  <- Example dependencies
+```
+
+### Installing Dependencies
+
+Project dependencies (such as `tensorflow` and `flwr`) are defined in `pyproject.toml` and `requirements.txt`. We recommend [Poetry](https://python-poetry.org/docs/) to install those dependencies and manage your virtual environment ([Poetry installation](https://python-poetry.org/docs/#installation)) or [pip](https://pip.pypa.io/en/latest/development/), but feel free to use a different way of installing dependencies and managing virtual environments if you have other preferences.
+
+#### Poetry
+
+```shell
+poetry install
+poetry shell
+```
+
+Poetry will install all your dependencies in a newly created virtual environment. To verify that everything works correctly you can run the following command:
+
+```shell
+poetry run python -c "import flwr"
+```
+
+If you don't see any errors you're good to go!
+
+#### pip
+
+Write the command below in your terminal to install the dependencies according to the configuration file requirements.txt.
+
+```shell
+pip install -r requirements.txt
+```
```

Comment:
We are gradually removing Poetry from the examples and only use a single `pyproject.toml` (i.e. an no `requirementes.txt`) to define the dependencies. See how we have done it in [examples/quickstart-pytorch](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch#installing-dependencies).

Reply:
I changed the version of flwr to 1.8.0 and of flwr-datasets to 0.1.0. Additionally, following the example, I changed the README to remove the Poetry dependence.

PS: Check the new [PR #3633](https://github.com/adap/flower/pull/3633) with all the modifications.